test/acceptance/tests/connect/connect_inject_test.go

```diff
@@ -108,8 +108,8 @@ func checkConnection(t *testing.T, options *k8s.KubectlOptions, client kubernete
 			require.NoError(r, err)
 			require.Contains(r, output, "hello world")
 		} else {
-			require.Error(t, err)
-			require.Contains(t, output, "503 Service Unavailable")
+			require.Error(r, err)
```

Comment:
you're gonna give me conflicts aren't you :D

Reply:
Sorry! but I think this was making tests fail in CI, so thank you!!!